src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/types/ValidateApplicantForSharingRequest.java

```diff
@@ -0,0 +1,41 @@
+package ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.types;
+
+
+/**
+ * 
+ * model class for request messages for /validateApplicantForSharing requests
+ * 
+ * @author archana
+ *
+ */
+public class ValidateApplicantForSharingRequest {
+	
+	private String applPartyId; 
+	
+	private String jurisdictionType;
+	
+	public ValidateApplicantForSharingRequest(String applPartyId, String jurisdictionType) {
+		super();
+		this.applPartyId = applPartyId;
+		this.jurisdictionType = jurisdictionType;
+	}
+	
+	public String getApplPartyId() {
+		return applPartyId;
+	}
+
+	public void setApplPartyId(String applPartyId) {
```

Comment:
Let's remove the setters from this class, thanks.

Reply:
removed the setters.